dvc/state.py

```diff
@@ -122,10 +126,10 @@ def get(self, path_info, fs):
 
         value = self.md5s.get(inode)
 
-        if not value or value[0] != mtime or value[1] != size:
+        if not value or value[0] != mtime or value[1] != str(size):
             return None
 
-        return HashInfo("md5", value[2], size=int(size))
+        return HashInfo("md5", value[2], size=size)
```

Comment:
This breaks backward compatibility for this db :slightly_frowning_face:

Reply:
I'll adjust it so that we go back to writing size as a string in the state DB.

The main thing here was that I don't think the `utils` function should have been returning both as strings (the returned `mtime` does have to be a string since for dirs it's actually the hash of all the file mtimes, but that doesn't apply to `size`)

Reply:
@pmrowla Agreed, that's part of an old code that used to be directly in old state class 🙁